app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -0,0 +1,193 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Heatmap from "~/components/visualizations/heatmap/Heatmap";
+import { getAMRCounts } from "~/api/amr";
+
+import cs from "./amr_heatmap_vis.scss";
+
+const METRIC_ALLELES = "alleles";
+const METRIC_GENES = "genes";
```

Comment:
It's awesome that you have these constants here. (It might make sense to do something similar with "coverage" vs "depth".

I think this might be better named as `VIEW_LEVEL_ALLELE` and `VIEW_LEVEL_GENE`. Since coverage and depth are the "metrics".

Reply:
Thanks for catching that!